rtengine/imagedata.h

```diff
@@ -64,85 +52,35 @@ class FrameData final
     int rating;
     std::string lens;
     IIOSampleFormat sampleFormat;
-
-    // each frame has the knowledge of "being an"
-    // or "being part of an" HDR or PS image
     bool isPixelShift;
     bool isHDR;
 
 public:
-
-    FrameData (rtexif::TagDirectory* frameRootDir, rtexif::TagDirectory* rootDir, rtexif::TagDirectory* firstRootDir);
-    virtual ~FrameData ();
-
-    bool getPixelShift () const;
-    bool getHDR () const;
-    std::string getImageType () const;
-    IIOSampleFormat getSampleFormat () const;
-    rtexif::TagDirectory* getExifData () const;
-    procparams::IPTCPairs getIPTCData () const;
-    static procparams::IPTCPairs getIPTCData (IptcData* iptc_);
-    bool hasExif () const;
-    bool hasIPTC () const;
-    tm getDateTime () const;
-    time_t getDateTimeAsTS () const;
-    int getISOSpeed () const;
-    double getFNumber () const;
-    double getFocalLen () const;
-    double getFocalLen35mm () const;
-    float getFocusDist () const;
-    double getShutterSpeed () const;
-    double getExpComp  () const;
-    std::string getMake () const;
-    std::string getModel () const;
-    std::string getLens () const;
-    std::string getSerialNumber () const;
-    std::string getOrientation () const;
-    int getRating () const;
+    FramesData(const Glib::ustring& fname);
```

Comment:
```suggestion
    explicit FramesData(const Glib::ustring& fname);
```
https://github.com/Beep6581/RawTherapee/blob/33a4737c89d9b7450532908d5ef6477d47a5f08e/rtengine/imagedata.h#L114

Reply:
Done in #6631